src/init.cpp

```diff
@@ -520,6 +520,7 @@ void SetupServerArgs(NodeContext& node)
         ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
     argsman.AddArg("-debugexclude=<category>", strprintf("Exclude debugging information for a category. Can be used in conjunction with -debug=1 to output debug logs for all categories except one or more specified categories."), ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
     argsman.AddArg("-logips", strprintf("Include IP addresses in debug output (default: %u)", DEFAULT_LOGIPS), ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
+    argsman.AddArg("-logmessages", "Log all p2p messages to file", ArgsManager::ALLOW_BOOL | ArgsManager::DEBUG_ONLY, OptionsCategory::DEBUG_TEST);
```

Comment:
I think the help should specify the file location dir and mention the location of and need to use the `contrib/message-logging/message-logging-parser.py` script.

Perhaps also add a markdown documentation file somewhere relevant that makes sense.

All of this, along with usage info, might also usefully be in the header docstring of the parser script.

Reply:
The docs could also helpfully mention that passing both send and recv files to the parser interleaves them.